libs/ui/src/setup_card_reader_page.tsx

```diff
@@ -1,23 +1,15 @@
+import { ERROR_SCREEN_MESSAGES } from './error_boundary';
 import { Main } from './main';
 import { Screen } from './screen';
-import { H1, P } from './typography';
-
-interface Props {
-  usePollWorkerLanguage?: boolean;
-}
-
-export function SetupCardReaderPage({
-  usePollWorkerLanguage = true,
-}: Props): JSX.Element {
-  const connectMessage = usePollWorkerLanguage
-    ? 'Please ask a poll worker to connect card reader.'
-    : 'Please connect the card reader to continue.';
+import { Caption, H1, P } from './typography';
 
+export function SetupCardReaderPage(): JSX.Element {
   return (
     <Screen>
       <Main centerChild>
         <H1>Card Reader Not Detected</H1>
-        <P>{connectMessage}</P>
+        <P>{ERROR_SCREEN_MESSAGES.RESTART}</P>
+        <Caption>{ERROR_SCREEN_MESSAGES.REACH_OUT}</Caption>
```

Comment:
SLI specifically flagged this screen as a screen that should include a reach-out message like the unknown error screens since there's no other action the user can take other than rebooting.